PrincessPaperplane/discordBot.py

```diff
@@ -77,6 +83,10 @@ def log(text):
         log("Exception in log: " + str(e))
         pass
 
+def add_commands():

```

Comment:
Scheinbar nicht nötig, da `discord.errors.ClientException: Command cmd_rank is already registered.` geworfen wird